resources/views/highlights/results.blade.php

```diff
@@ -7,6 +7,22 @@
 @section('content')
 
 <h2>Search results</h2>
+ {{-- If search results lead to social media websites, don't include them in the final results number --}}
+ {{-- As they're set not to show up in the result cards --}}
+@if(!empty($records))
+    @foreach($records as $result)
+    @php
+        $url = $result['url'][0];
+        $is_social = false;
+        if(str_contains($url, 'x.com') || str_contains($url, 'facebook.com') || str_contains($url, 'linkedin.com') || str_contains($url, 'youtube.com') || str_contains($url, 'instagram.com') ) {
+            $is_social = true;
+        }
+        if($is_social === true) {
+            $number = $number -= 1;
```

Comment:
This is a mixture of using the short and long ways of doing this. Either it should be:

```$number = $number - 1;```

Or:

```$number -= 1;```

I'm not 100% sure if the code would work correctly as it currently is (although I think it should do). Regardless of that, it should be changed to either of the above options.

See:
https://www.php.net/manual/en/language.operators.assignment.php#language.operators.assignment.arithmetic

Reply:
That's a good point! Good spot!